tanner/create_tables.py

```diff
@@ -0,0 +1,71 @@
+import asyncio
+
+
+class CreateTables:
+    @staticmethod
+    async def create_data_tables(pg_client):
+        """Create all the required tables in
+            the postgres database
+
+        Arguments:
+            pg_client {aiopg.pool.Pool}
+        """
+        async with pg_client.acquire() as conn:
+            async with conn.cursor() as cur:
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "paths" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "path" TEXT NOT NULL,
+                    "timestamp" TIMESTAMP NOT NULL,
+                    "response_status" INT NOT NULL,
+                    "attack_type" INT NOT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "cookies" (
+                    "sess_uuid" UUID PRIMARY KEY NOT NULL UNIQUE,
+                    "key" TEXT NULL,
+                    "value" TEXT NULL
+                )
+                """
+                )
+
+                await cur.execute(
+                    """
+                CREATE TABLE IF NOT EXISTS "session_data" (
+                    "sess_uuid" UUID NOT NULL PRIMARY KEY,
+                    "snare_uuid" TEXT NOT NULL,
+                    "peer.ip" INET NOT NULL,
+                    "peer.port" INT NOT NULL,
+                    "location.country" TEXT NULL,
+                    "location.country_code" TEXT NULL,
+                    "location.city" TEXT NULL,
+                    "location.zip_code" INT NULL,
+                    "user_agent" TEXT NOT NULL,
+                    "start_time" TIMESTAMP,
```

Comment:
use `now()` as default. start and end should not be null.

Reply:
if we use NOW() for both start and end then do we need to provide the `NOT NULL`? Asking since now would always provide the value so it won't be null right?

Reply:
Correct, with DEFAULT you don't need NOT NULL except if you want to prevent the client from explicitly setting it to null.
Not sure if we want to implicitly set the end time by using `DEFAULT now()` or have it NULL until we explicitly end the session. What do you think?

Reply:
I don't see a problem in any of the cases.